include/mega/win32/megafs.h

```diff
@@ -139,7 +140,6 @@ class MEGA_API WinFileAccess : public FileAccess
     bool fopen(string*, bool, bool, bool);
     void updatelocalname(string*) override;
     bool fread(string *, unsigned, unsigned, m_off_t);
-    bool frawread(byte *, unsigned, m_off_t);
```

Comment:
these classes should always override the four virtual implementation functions?

Reply:
frawread is not overriden if you look at the cpp of this class. frawread is not even virtual. It was an oversight to paste it here.